hs_core/models.py

```diff
@@ -961,6 +961,36 @@ def __unicode__(self):
         return self.value
 
 
+class FundingAgency(AbstractMetaDataElement):
+    term = 'FundingAgency'
+    agency_name = models.TextField(null=False)
+    award_title = models.TextField(null=True, blank=True)
+    award_number = models.TextField(null=True, blank=True)
+    agency_url = models.URLField(null=True, blank=True)
+
+    class Meta:
+        unique_together = ("content_type", "object_id", "award_number", "agency_name")
```

Comment:
It would be good to just make sure users don't create what look like duplicate records. For example, it would appear as a duplicate if they create two awards from the same funding agency, but don't distinguish with either award numbers or award names. Simple logic to prevent this would be good.